cmd/capibmadm/clients/powervs/client.go

```diff
@@ -0,0 +1,19 @@
+package powervs
+
+import (
+	"fmt"
+
+	"github.com/IBM-Cloud/power-go-client/ibmpisession"
+	
+	"sigs.k8s.io/cluster-api-provider-ibmcloud/cmd/capibmadm/clients/iam"
+)
+
+func CreatePowerVSSession(accountID string, region string, zone string, debug bool) (*ibmpisession.IBMPISession, error) {
```

Comment:
lets see if we can get rid of package name from the functions, can be renamed to `NewPISession`, applies to all the other `New` methods as well..

Reply:
Renamed the funcs.